docs/cluster_spec.md

```diff
@@ -45,7 +45,7 @@ AutomaticReplacementOptions controls options for automatically replacing failed
 | ----- | ----------- | ------ | -------- |
```

Comment:
All changes in this file must be done in foundationdbcluster_types.go otherwise they will be removed the next time.